projects/addon-mobile/directives/dropdown-mobile/dropdown-mobile.component.ts

```diff
@@ -8,15 +8,15 @@ import {
 } from '@angular/core';
 import {IntersectionObserverModule} from '@ng-web-apis/intersection-observer';
 import {TuiKeyboardService} from '@taiga-ui/addon-mobile/services';
-import type {TuiSwipe} from '@taiga-ui/cdk';
+import type {TuiSwipeEvent} from '@taiga-ui/cdk';
```

Comment:
Add comment to

https://github.com/taiga-family/taiga-ui/issues/5808#issuecomment-2172558410